app/components/long_reads/client.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module LongReads
+  # HTTParty client for LongReads that takes an article path
+  # and get's the text of the article
+  class Client < GenericClient
```

Comment:
Assumes too much for instance variable '@article_id' (InstanceVariableAssumption)